pkgs/by-name/up/updfparser/package.nix

```
@@ -6,13 +6,12 @@
 
 stdenv.mkDerivation rec {
   name = "updfparser";
-  version = "unstable-2023-08-08";
-  rev = "c5ce75b9eea8ebb2746b13eeb0f335813c615115";
+  version = "unstable-2024-03-24";
```

```suggestion
  version = "0-unstable-2024-03-24";
```
Since recently all versions must start with a digit. In case the package never had a stable release, `0` is preferred